projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/content-page-meta.resolver.migration.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  PAGE_META_SERVICE,
+  PRODUCT_PAGE_META_RESOLVER,
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const PRODUCT_PAGE_META_RESOLVER_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: PRODUCT_PAGE_META_RESOLVER,
```

Comment:
I think it was supposed to be for `ContentPageMetaResolver`?

Also there are more protected proerties that were removed: `cms$`, `breadcrumbs$` and `title$`.

Reply:
correct, thanks